From: Balint Reczey <balint.reczey@canonical.com>
Date: Thu, 9 Jul 2020 12:50:22 +0200
Subject: Always build with -no-undefined
Forwarded: not-needed

---
 Makefile.am | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 2181500..14ac0b9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -251,13 +251,13 @@ EXTRAS_SRC =					\
 	evrpc.c					\
 	http.c
 
-if BUILD_WITH_NO_UNDEFINED
+#if BUILD_WITH_NO_UNDEFINED
 NO_UNDEFINED = -no-undefined
 MAYBE_CORE = libevent_core.la
-else
-NO_UNDEFINED =
-MAYBE_CORE =
-endif
+#else
+#NO_UNDEFINED =
+#MAYBE_CORE =
+#endif
 
 AM_CFLAGS = $(LIBEVENT_CFLAGS)
 AM_CPPFLAGS = -I$(srcdir)/compat -I./include -I$(srcdir)/include $(SYS_INCLUDES) $(LIBEVENT_CPPFLAGS)
